core/test-api/db/src/test/java/org/opennms/core/test/db/MigratorIT.java

```diff
@@ -73,23 +80,29 @@
 @ContextConfiguration(locations={
         "classpath:/migratorTest.xml"
 })
-@JUnitTemporaryDatabase
-public class MigratorIT {
+@JUnitTemporaryDatabase(createSchema = false)
+public class MigratorIT implements TemporaryDatabaseAware<TemporaryDatabase> {
     private static final Logger LOG = LoggerFactory.getLogger(MigratorIT.class);
 
     @Autowired
     DataSource m_dataSource;
 
-    @Autowired
-    ResourceLoader m_resourceLoader;
-
     @Autowired
     ApplicationContext m_context;
 
+    TemporaryDatabase m_database;
+
+    @Override
+    public void setTemporaryDatabase(TemporaryDatabase database) {
+        m_database = database;
+    }
+
     @Before
     public void setUp() throws Exception {
         System.setProperty("org.apache.logging.log4j.simplelog.StatusLogger.level", "INFO");
         MockLogAppender.setupLogging();
+
+        TemporaryDatabasePostgreSQL temp = new TemporaryDatabasePostgreSQL();
```

Comment:
AFAICT this does nothing, or are you doing it just for initialization?  If so you don't need the variable.